See blog/see-blog.css

```diff
@@ -1,6 +1,210 @@
-@import url('https://fonts.googleapis.com/css2?family=Metrophobic&family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap');
+@import url("https://fonts.googleapis.com/css2?family=Metrophobic&family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap");
 
 * {
-    font-family: 'Metrophobic', sans-serif;
-    font-family: 'Poppins', sans-serif;
-}
\ No newline at end of file
+  font-family: "Metrophobic", sans-serif;
+  font-family: "Poppins", sans-serif;
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 2px;
+  height: 50px;
+}
+.left-side {
+  width: 24px;
+  height: 24px;
+  left: 40px;
+  top: 74px;
+  margin-left: 16px;
+}
+.right-side {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-right: 16px;
+}
+img {
+  padding: 4px;
+  margin-top: 5px;
+  width: 30px;
+  height: 30px;
+}
+.more {
+  width: 24px;
+  height: 24px;
+  left: 359px;
+  top: 50px;
+  margin-top: 4px;
+  padding-left: 10px;
+}
+.top-par {
+  font-family: Metrophobic, bold;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 150%;
+  width: 305px;
+  height: 50px;
+  left: 16px;
+  top: 94px;
+  margin-left: 16px;
+  margin-top: 20px;
```

Comment:
There is too much space between the `header` and the `top-par` section, so you can delete `margin-top`